integration_tests/test_snapcraft_recording.py

```diff
@@ -45,6 +45,7 @@ def test_prime_with_architectures(self):
         This snap doesn't have stage or build packages and is declared that it
         works on all architectures.
         """
+
```

Comment:
When the name of the test is clear, they are not needed. But here I couldn't fit good names in 80 chars.
The names are confusing, because with architecture could mean that the source has arch, or that the record has arch. Without dependencies could mean that the packages have no dependencies, or that all the dependencies are declared. I welcome suggestions for better test names.